### PR TITLE
Init compose

### PR DESCRIPTION
**Pull request by KyuubiRan**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext{
-        kotlin_version = "1.6.10"
```
Comment (NextAlone) [inline build.gradle:4]: 还不支持1.6.10吗？

**Review by NextAlone — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (KyuubiRan) [reply build.gradle:4]: 嗯

**Comment by qwq233** [comment]
Can one of the admins verify this patch?

**Comment by keta1** [comment]
> Can one of the admins verify this patch?

no

**Comment by qwq233** [comment]
> > Can one of the admins verify this patch?
> 
> no

草 我看看能不能给关掉

**Comment by keta1** [comment]
> > > Can one of the admins verify this patch?
> > 
> > no
> 
> 草 我看看能不能给关掉

关了不好玩

**Comment by qwq233** [comment]
> > > > Can one of the admins verify this patch?
> > > 
> > > 
> > > no
> > 
> > 
> > 草 我看看能不能给关掉
> 
> 关了不好玩

= =那我看看能不能设置成Github App 这是拿我PAT做的玩的 我会收到notification orz

**Comment by jiasswee** [comment]
@qwq233 ???

**Comment by qwq233** [comment]
> @qwq233 ???

？

**Pull request merged**